src/test.rs

```diff
@@ -1,3 +1,5 @@
+// Copyright 2019 Centrality Investments Limited
```

Comment:
Could u add the rest of the license header 😄

Reply:
There's no OSS license for doughnuts atm, just copyright